### PR TITLE
Include url of asset on serialization

### PR DESCRIPTION
**Pull request by ben181231**
connect #64

### CODE REVIEW

**Comment by oursky-travis** [comment]
@ben181231, thanks for your PR! By analyzing the history of the files in this pull request, we identified @cheungpat and @rickmak to be potential reviewers.

File: Example/Tests/SKYRecordSerializerTests.m [hunk 1]
```diff
@@ -74,6 +74,7 @@
             expect(dictionary[@"asset"]).to.equal(@{
                 @"$type" : @"asset",
                 @"$name" : @"asset-name",
+                @"$url" : @"http://ourd.test/files/asset-name",
```
Comment (cheungpat) [inline Example/Tests/SKYRecordSerializerTests.m]: why suddenly `ourd`?

Comment (ben181231) [inline Example/Tests/SKYRecordSerializerTests.m]: Oh... It should be `skygear`.

Let me rebase this PR and correct it.

**Comment by ben181231** [comment]
PR updated.

**Pull request merged**